src/sagemaker/jumpstart/artifacts.py

```diff
@@ -369,7 +371,8 @@ def _retrieve_default_environment_variables(
     region: Optional[str] = None,
     tolerate_vulnerable_model: bool = False,
     tolerate_deprecated_model: bool = False,
-):
+    use_case: EnvVariableUseCase = EnvVariableUseCase.AWS_SDK,
```

Comment:
Adding `EnvVariableUseCase` seems a bit strange, it is rare to see a `*UseCase` type defined in most implementations. Also, `AWS_SDK` seems out-of-place in the SageMaker Python SDK. 

In the interest of unblocking the main goal of the new JumpStart classes without stepping through one-way doors, could we instead implement a new internal function `_env_variables_required_for_model_class(env_vars: Dict[str, str])` which the `JumpstartModel` class will use?